### PR TITLE
Feature/thorchain node ip removal

### PR DESCRIPTION
**Pull request by mbartn**
* Remove usage of THORCHAIN_NODE_IP from the code and docs
* change of mocking server - instead of manually setting endpoint in code using `if(debug)` mock api was adapted to handle requests in the same way like thorchain does, e.g. `http://localhost:26657/status`

### CODE REVIEW

File: helpers.py [hunk 1]
```diff
@@ -326,50 +324,17 @@ def is_binance_node_healthy():
...
 
     if DEBUG:
-        return 'http://localhost:8000/nodeaccounts.json'
-    elif THORCHAIN_NODE_IP:
-        return 'http://' + THORCHAIN_NODE_IP + ':1317/thorchain/nodeaccounts'
+        return 'http://localhost:8080/nodeaccounts.json'
     else:
         endpoints = requests.get('https://testnet-seed.thorchain.info').json()
         random_endpoint = endpoints[random.randrange(0, len(endpoints))]
         return 'http://' + random_endpoint + ':1317/thorchain/nodeaccounts'
```
Comment (mbartn) [inline helpers.py:335]: @rekt0x @mr-rooftop please pay attention to this change. Is it safe to always return random enpoint here?

File: helpers.py [hunk 2]
```diff
@@ -206,10 +206,11 @@ def add_thornode_to_user_data(user_data, address, node):
             break
 
     user_data['nodes'][address] = {}
-    user_data['nodes'][address]['alias'] = alias
     user_data['nodes'][address]['status'] = node['status']
     user_data['nodes'][address]['bond'] = node['bond']
     user_data['nodes'][address]['slash_points'] = node['slash_points']
+    user_data['nodes'][address]['ip_address'] = node['ip_address']
```
Comment (rekt0x) [inline helpers.py:212]: Make sure `node['ip_address']` gets sanitized here, so you don't have to call `.rstrip('/')` all the time below.

File: helpers.py (hunk 1 above)
Comment (rekt0x) [reply helpers.py:335]: Yeah, that is intended. We want to talk to the network and therefore we connect to a random node of the network (so the bot does not always use the same node to enter the network).

**Review by rekt0x — CHANGES_REQUESTED** [review]

File: helpers.py (hunk 2 above)
Comment (mbartn) [reply helpers.py:212]: good point!

**Review by rekt0x — APPROVED** [review]

**Review by rekt0x — APPROVED** [review]

**Review by mr-rooftop — APPROVED** [review]

**Pull request merged**